packages/edit-site/src/plugins/index.js

```diff
@@ -17,6 +19,15 @@ registerPlugin( 'edit-site', {
 				<ToolsMoreMenuGroup>
 					<SiteExport />
 					<WelcomeGuideMenuItem />
+					<MenuItem
+						href={ __(
+							'https://wordpress.org/support/article/site-editor/'
```

Comment:
We should add the `external` icon here as well along with  the `VisuallyHidden` with `opens in a new tab` as in the post editor, no?

Reply:
Resolved in 711e6d0.